app/views/Project/PillarAnalysis/index.tsx

```diff
@@ -283,6 +391,15 @@ function PillarAnalysis(props: Props) {
                 analyticalStatements = [newAnalyticalStatement1, newAnalyticalStatement2];
             }
 
+            if ((response.analyticalStatements?.length ?? 0) > 0) {
+                const entryIds = response.analyticalStatements
+                    ?.map((statement) => statement.analyticalEntries)
+                    .flat()?.filter(isDefined).map((entry) => entry.entry);
+                if (entryIds) {
+                    setInitialEntries(entryIds);
+                }
+            }
```

Comment:
Any reason this was added?

Reply:
Discussed on call to remove intialEntries state